### ISSUE TITLE
Spoiler in part of sentence

### ISSUE THREAD

**Issue by Xeunyx-Cypher** [LemmyNet/lemmy#2321]
Markdown feature to add spoiler in part of sentence (some words or more) like on reddit, the spoiler appears as a grayed block where you press to reveal it :
 
![IMG_20220616_103030](https://user-images.githubusercontent.com/44645135/173991586-0a4fd7ff-57d3-4e60-b297-cb850b3c350c.jpg)

**Comment by DMTryptamines** [comment]
Lemmy has a spoiler feature already although it works slightly different, it's the exclamation in a triangle highlighted in yellow 
![lemmyspoiler](https://user-images.githubusercontent.com/86132393/174083669-d747784a-bf81-449d-8021-c930777cb6e2.PNG)

**Comment by Xeunyx-Cypher** [comment]
Yes I know. But it doesn't have a option to mark a word as spoiler in sentence without breaking the entire line. A simple spoiler markdown symbol like >! would be much better. 

What I wanted something to look like :

![IMG_20220618_172523](https://user-images.githubusercontent.com/44645135/174435540-04b6493e-ae98-43a4-b6c0-1a6b40d245d9.jpg)

And ended up looking like :

![IMG_20220618_173028](https://user-images.githubusercontent.com/44645135/174435602-ac71621d-55bc-48d9-aa60-5d96f8fa819b.jpg)

**Comment by dessalines** [comment]
It looks like there are now some markdown-it spoiler plugins: https://github.com/search?q=markdown+it+spoiler

I'd accept a PR for one. They should support multi-line spoilers tho, like the current one does.
